autorest/codegen/models/enum_schema.py

```diff
@@ -84,20 +84,12 @@ def serialization_type(self) -> str:
     def type_annotation(self) -> str:
         """The python type used for type annotation
 
-        :return: The type annotation for this schema
-        :rtype: str
-        """
-        return f'Union[{self.enum_type.type_annotation}, "{self.name}"]'
-
-    @property
-    def operation_type_annotation(self) -> str:
-        """The python type used for type annotation
-
         :return: The type annotation for this schema
         :rtype: str
         """
         return f'Union[{self.enum_type.type_annotation}, "_models.{self.name}"]'
 
+
```

Comment:
unnecessary new line?

Reply:
i can remove, it's not triggering pylint though